### PR TITLE
updated more functions that should be supported by qemu guest agent

### PR DESCRIPTION
**Pull request by Zhengtong**
add guest-sync-delimited guest-set-user-password guest-get-vcpus guest-set-vcpus command  support

id:1328001 1327995 1327990

### CODE REVIEW

**Comment by PandaWei** [comment]
LGTM

File: virttest/guest_agent.py [hunk 1]
```diff
@@ -497,19 +502,56 @@ def shutdown(self, mode=SHUTDOWN_MODE_POWERDOWN):
...
+        return self.cmd(cmd=cmd, args=args)
+
+    @error_context.context_aware
+    def get_vcpus(self):
+        """
+        Get the vcpus infomation
+        """
+        cmd = "guest-get-vcpus"
+        if not self._has_command(cmd):
+            return False
```
Comment (CongLi) [inline virttest/guest_agent.py]: I think it's better to raise error here.

Comment (Zhengtong) [inline virttest/guest_agent.py]: I handle the error in the upper layer (in tp-qemu).  In my opinion, for API, it's better to just give a result.   I add the error handler in https://github.com/autotest/tp-qemu/pull/623

Comment (will-Do) [inline virttest/guest_agent.py]: Agree

**Comment by will-Do** [comment]
LGTM

**Comment by CongLi** [comment]
Acked-by: Cong Li

**Pull request merged**